homeassistant/components/totalconnect/config_flow.py

```diff
@@ -88,6 +89,14 @@ async def async_step_locations(self, user_entry=None):
                     },
                 )
         else:
+            # Force the loading of locations using I/O
+            number_locations = await self.hass.async_add_executor_job(
+                self.client.get_number_locations,
+            )
+            if number_locations < 1:
+                raise HomeAssistantError(
```

Comment:
Show an error in the form or abort the flow.

Does this addition need to be done in this PR?

Reply:
I can split it to a second PR if that's better, but it is needed to fix the problem preventing someone from adding the integration via config flow.

Reply:
Please do unrelated fixes separately. This PR should only be about bumping the library and any changes in the integration to make it compatible with the new library version.

Reply:
I've reverted the changes related to config flow and with submit a separate PR shortly